### ISSUE TITLE
Missing "Authorize resources"

### ISSUE THREAD

**Issue by Sonesson** [MicrosoftDocs/azure-devops-docs#4643]
There is no button for "Authorize resources" available/visible when I follow your steps to authorize a pipeline. Tried both Edge and Firefox. The red box is only visible from Edit-&gt;Queue dialog. Yes, I am a User and creator of the resource.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ee4ec9d0-e0d5-4fb4-7c3e-b84abfa290c2
* Version Independent ID: 3e2b80d9-30e5-0c48-49f0-4fcdfedf5eee
* Content: [Resources - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/resources?view=azure-devops#troubleshooting-authorization-for-a-yaml-pipeline)
* Content Source: [docs/pipelines/process/resources.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/pipelines/process/resources.md)
* Product: **devops**
* Technology: **devops-cicd**
* GitHub Login: @vtbassmatt
* Microsoft Alias: **macoope**

**Comment by jmlane** [comment]
![image](https://user-images.githubusercontent.com/95528/63976954-e356f280-ca88-11e9-9fa7-995d3b76b8e5.png)

Was this lost in a Pipelines UI update perhaps?

**Comment by vijayma** [comment]
This is not expected. We will look into this. Are you still able to "Run" the pipeline. The failed run will have the authorize button on it, if you are able to get that far.

**Comment by jmlane** [comment]
No, the run is blocked by the error message shown in the screenshot.

I tried setting the variable group to allowed for all pipelines, running,
and then disabling that flag to see if I could get a different result. The
UI flow is the same as shown here.

**Comment by michaelwestbrook** [comment]
Just want to throw my 2¢ here. I was at least able to unblock myself by setting a temporary branch policy that runs my build definition as a pull request policy. Then I created a new pull request targeting my branch that has the resource I want to authorize. That started a build (that failed immediately due to unauthorized resources) and from there I was able to see the 'authorize resources' button on the summary page. This should be fine as a temporary work around.

**Comment by GrimGadget** [comment]
This is still broken.

In order to get the authorize button, you have to trigger a build or submit a PR, which will fail, and then you get the button to click.  But attempting to manually run a build gives the result in the screenshot above without ever actually queuing a build, so you never get the build output screen, where the button is located.

**Comment by GrimGadget** [comment]
I'm not sure what actually resets the authorization status on the resources, so it's possible/likely that editing the triggers in the build definition is not a viable solution, either.

**Comment by gitfool** [comment]
I had to unlink the group variables to get past this and it was not easy to find where to do that in the UI! I've since abandoned group variables.

**Comment by GrimGadget** [comment]
That seems related.  I encountered this problem after I set up a variable group for SonarCloud key/project/version info so I would only have one place to change it for the project (which has three pipelines)

**Comment by LanceMcCarthy** [comment]
I had to abandon Group linked variables as well. It never recognizes them. This seems to also be a problem for SecureFiles, even if a SecureFile is allowed for all pipelines.

**Comment by qchero** [comment]
As people said, trigger a run by Pull Request instead of manual steps will review a "Authorize Resource" button. I was able to be unblocked by that.

**Comment by cxwtool** [comment]
This issue hasn't been updated in more than 180 days, so we've closed it. If you feel the issue is still relevant and needs fixed, please reopen it and we'll take another look. We appreciate your feedback and apologize for any inconvenience.
